UPGRADE-1.11.md

```diff
@@ -47,6 +47,10 @@ Add a new bundle to your list of used bundles in `config/bundles.php` if they ar
     +   Sylius\Calendar\SyliusCalendarBundle::class => ['all' => true],
     ```
 
+### Order prices recalculator
+
+Service `Sylius\Component\Core\Calculator\ProductVariantPriceCalculatorInterface` has been deprecated. Use `Sylius\Component\Core\Calculator\ProductVariantPricesCalculatorInterface` instead.
```

Comment:
This service has been deprecated in Sylius 1.8, so the proper info should be in UPGRADE-1.8 from 1.8 version. You should add in this PR, that 
>'Passing a "Sylius\Component\Core\Calculator\ProductVariantPriceCalculatorInterface" to Sylius\Component\Core\OrderProcessing\ OrderPricesRecalculator" constructor is deprecated since Sylius 1.11 and will be prohibited in 2.0. Use "Sylius\Component\Core\Calculator\ProductVariantPricesCalculatorInterface" instead.